### ISSUE TITLE
Dataset issues

### ISSUE THREAD

**Issue by chenyuaner** [hspark1212/MOFTransformer#173]
Dear authors,

Thank you for developing and sharing this great model and data. Your work on COF materials for gas adsorption is truly inspiring.

I'm particularly interested in reproducing your pretraining and fine-tuning procedures for methane adsorption in COFs at high pressure conditions. I have found most of the data from other sources and your published GitHub repository, but I couldn't locate the specific data labels for methane uptake at 65 bar.
If possible, could you please share the pretraining data label for methane uptake at 65 bar via email？My email address is: 914367679@qq.com
Thank you very much for your consideration and for your contributions to the field.
